### DISCUSSION TITLE
How do I install and use the blog plugin?

### DISCUSSION THREAD

**Discussion by pivic** [mkdocs/mkdocs#3173] [General]
I just cloned the `mkdocs-material-insiders` repo and tried to use the blog plugin, but I see this error when I try to start the live preview server:

> mkdocs serve
> ERROR    -  Config value 'plugins': The "blog" plugin is not installed
> Aborted with 1 Configuration Errors!

I've tried to find info on how to install and use the plugin but to no avail.

I'd appreciate any help! Thanks in advance.

**Comment by squidfunk** [comment]
Which [installation method](https://squidfunk.github.io/mkdocs-material/insiders/getting-started/#installation) did you use? We recommend using [`pip`](https://squidfunk.github.io/mkdocs-material/insiders/getting-started/#with-pip), which sets up everything accordingly:

```
pip install git+https://${GH_TOKEN}@github.com/squidfunk/mkdocs-material-insiders.git
```

Also make sure you're inside the right environment, i.e., that `pip list` includes `mkdocs-material-x.x.x+insiders-x.x.x`.

**Reply by squidfunk** [reply]
Additionally, please don't post questions regarding Insiders here in the future – this is MkDocs, an upstream dependency of Material for MkDocs. [Please use our own discussion board](https://github.com/squidfunk/mkdocs-material/discussions).

**Reply by pivic** [reply]
Thanks! I've moved my discussion: https://github.com/squidfunk/mkdocs-material/discussions/5333
